site/content/tutorial/04-logic/05-keyed-each-blocks/app-a/Thing.svelte

```diff
@@ -6,19 +6,14 @@
 	const initial = current;
 </script>
 
-<p>
-	<span style="background-color: {initial}">initial</span>
-	<span style="background-color: {current}">current</span>
-</p>
+<p>initial: {initial}, current: {current}</p>
```

Comment:
```suggestion
<p>
  <span>initial: {initial}</span>
  <span>current: {current}</span>
</p>
```

I find the initial design of the tutorial nicer, more appealing and friendly

Reply:
Do you want to go back to two boxes instead of one? The reason I made one was because it visually identifies one component instance as one element. I found it confusing that the props corresponded to boxes instead of the components themselves.

I agree though that the current grey bar is lame and we can do better. I'm open to suggestions and will try to come up with a better styling.